python/python/lance/fragment.py

```diff
@@ -384,3 +385,76 @@ def metadata(self) -> FragmentMetadata:
         FragmentMetadata
         """
         return FragmentMetadata(self._fragment.metadata().json())
+
+
+def write_fragments(
+    data: ReaderLike,
+    dataset_uri: Union[str, Path],
+    schema: Optional[pa.Schema] = None,
+    *,
+    max_rows_per_file: int = 1024 * 1024,
+    max_rows_per_group: int = 1024,
+    max_bytes_per_file: int = 90 * 1024 * 1024 * 1024,
```

Comment:
How do we come up with `90` here.

Reply:
There's a 100GB limit. `max_bytes_per_file` is a soft limit so it has a bit of a buffer for now. Could make it smaller if we wanted, but I figured 90 is plenty for most use cases anyways.

Reply:
Let's add a comment to describe the logic here?